devops/binary-footprint-baseline/baseline.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "distro": "ubuntu-18.04",
+    "arch": "x86_64",
+    "sizeBytes": 1772612,
```

Comment:
Are the `sizeBytes` here the expected value based on our current package size?

Reply:
no, these are just the current sizes. We store them as snapshots and we compare the produced local debian package, to the ones contained in the baseline